src/control/BarrettFingerKinematicSimulationSpreadCommandExecutor.cpp

```diff
@@ -104,53 +101,53 @@ std::future<void> BarrettFingerSpreadCommandExecutor::execute(
 
     mPromise.reset(new std::promise<void>());
 
-    if (_goalPosition < mDofLimits.first)
+    if (goalPosition < mDofLimits.first)
       mGoalPosition = mDofLimits.first;
-    else if (_goalPosition > mDofLimits.second)
+    else if (goalPosition > mDofLimits.second)
       mGoalPosition = mDofLimits.second;
     else
-      mGoalPosition = _goalPosition;
+      mGoalPosition = goalPosition;
 
-    mCollideWith = _collideWith;
     mInExecution = true;
 
     return mPromise->get_future();
   }
 }
 
 //=============================================================================
-void BarrettFingerSpreadCommandExecutor::step(double _timeSincePreviousCall)
+void BarrettFingerKinematicSimulationSpreadCommandExecutor::step(
+  const std::chrono::milliseconds& timeSincePreviousCall)
```

Comment:
Same comment as above. I suggest removing this parameter entirely, so this class complies with the `PositionCommandExecutor` interface.